lib/chime-notifier/chime-notifier.ts

```diff
@@ -40,33 +43,49 @@ export class ChimeNotifier extends Construct {
   constructor(scope: Construct, id: string, props: ChimeNotifierProps) {
     super(scope, id);
 
-    const message = props.message ?? "@All Pipeline '$PIPELINE' failed on '$REVISION' in '$ACTION' (see $URL)";
+    const message = props.message ?? "/md @All Pipeline **$PIPELINE** failed in action **$ACTION**. Latest change:\n```\n$REVISION\n```\n([Failure details]($URL))";
 
     if (props.webhookUrls.length > 0) {
-      const notifierLambda = new lambda.Function(this, 'Default', {
+      // Reuse the same Lambda code for all pipelines, we will move the Lambda parameterizations into
+      // the CloudWatch Event Input.
+      const notifierLambda = new lambda.SingletonFunction(this, 'Default', {
         handler: 'index.handler',
-        code: lambda.Code.inline(fs.readFileSync(path.join(__dirname, 'notifier-handler.js')).toString('utf8')),
+        uuid: '0f4a3ee0-692e-4249-932f-a46a833886d8',
+        code: lambda.Code.inline(stripComments(fs.readFileSync(path.join(__dirname, 'notifier-handler.js')).toString('utf8'))),
```

Comment:
why not just use an asset?

Reply:
Kinda just mirrored this off of the pipeline watcher which also uses inline code.